docs/dev/Pagination-v2.md

```diff
@@ -504,6 +504,8 @@ sequenceDiagram
     participant QueryPlanFactory
     participant QueryService
     participant OpenSearchExecutionEngine
+    participant LogicalCursor
+    participant DefaultImplementor
```

Comment:
These 2 are not used in the graph.
`CreatePagingScanBuilder` in the previous graph is also orphaned.

Reply:
That was an unfinished revision. It should be better now.

Reply:
Please remove `CreatePagingScanBuilder` in the previous graph